src/apache/conf/ssl.conf

```diff
@@ -161,3 +161,17 @@ SSLRandomSeed connect file:/dev/urandom 512
         Header always set Strict-Transport-Security "max-age=63072000; includeSubdomains;"
     </IfDefine>
 </VirtualHost>
+
+#########################
+#        Snapweb        #
+#########################
+Listen 4201
+<VirtualHost *:4201>
```

Comment:
Yep, I'd say it's not needed any more for most users if they can simply add the URL. Enterprise users might need to create their own vhost, but it's out of sope for this project.